docs_src/threat-models/secret-store/threat_model.md

```diff
@@ -12,12 +12,11 @@ and not the EdgeX project as a whole.
 
 ## Assumptions
 
-The EdgeX Framework is a API-based software framework that strives to be platform and architecture-independent. The threat model considers only the following two deployment scenarios:
+The EdgeX Framework is a API-based software framework that strives to be platform and architecture-independent. The threat model considers only the following deployment scenario:
 
 * A containerized implementation based on Docker.
-* A confined implementation based on Snaps.
 
-The threat model presented in this document analyzes the secret management subsystem of EdgeX, and has considerations for both of the above runtime environments, both of which implement protections beyond a stock user/process runtime environment.  In generic terms, the secret management threat model assumes:
+The threat model presented in this document analyzes the secret management subsystem of EdgeX, and has considerations for the above runtime environments, which implements protections beyond a stock user/process runtime environment.  In generic terms, the secret management threat model assumes:
```

Comment:
```suggestion
The threat model presented in this document analyzes the secret management subsystem of EdgeX, and has considerations for the above runtime environment, which implements protections beyond a stock user/process runtime environment.  In generic terms, the secret management threat model assumes:
```

Reply:
As suggested earlier, the 'threat_model' file will be kept as it is. The snap-related change is no longer needed.